src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -34,6 +34,13 @@ internal sealed class TupleTypeSymbol : WrappedNamedTypeSymbol
         private readonly ImmutableArray<string> _elementNames;
 
         /// <summary>
+        /// Which element names were inferred and therefore cannot be used.
+        /// If none of the element names were inferred, or inferred names can be used (no tracking necessary), leave as default.
+        /// This information is ignored in type equality and comparison.
+        /// </summary>
+        private readonly ImmutableArray<bool> _errorPositions;
```

Comment:
Since the `_errorPositions` field only applies to use of inferred names at those positions, would it make sense to call this array `_cannotUseInferredNames`?